generic/secrets/security/detected-artifactory-password.yaml

```diff
@@ -1,7 +1,10 @@
 rules:
 - id: detected-artifactory-password
-  pattern-regex: |-
-    AP[\dABCDEF][a-zA-Z0-9]{8,}
+  patterns:
+  - pattern-regex: |
+      \bAP[\dABCDEF][a-zA-Z0-9]{8,}
```

Comment:
how come this is unbounded `{8,}` @minusworld? Aren't the tokens one or two pre-defined lengths?

Reply:
I copied these from another source; I set the word boundary `\b` on them in this PR. I am uncertain of the original research that went into them. I propose waiting and seeing if the issue persists. If it does, I don't think the unbounded regexes are a good approach and we should nuke these or figure out what the correct lengths are.